Extension/src/Debugger/configurationProvider.ts

```diff
@@ -231,11 +231,61 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
                     config.pipeTransport.pipeProgram = replacedPipeProgram;
                 }
             }
+
+            const macOSMIMode: string = config.osx?.MIMode ?? config.MIMode;
+            const macOSMIDebuggerPath: string = config.osx?.miDebuggerPath ?? config.miDebuggerPath;
+
+            // Validate LLDB-MI
+            if (os.platform() === 'darwin' && // Check for macOS
+                (!macOSMIMode || macOSMIMode === 'lldb') &&
+                !macOSMIDebuggerPath && // User did not provide custom lldb-mi
+                !this.hasLLDBFramework()
+                ) {
+
+                const moreInfoMessage: string = localize("lldb.framework.more.info", "More Info");
+                const LLDBFrameworkMissingMessage: string = localize("lldb.framework.not.found", "Unable to locate 'LLDB.framework' for lldb-mi.");
```

Comment:
I feel like there should be a bit more information in this message.  I think users are trained (at least I am) to be disappointed in "More Info" promises and I'm worried there's not enough incentive in this message to convince them to click the button before giving up or opening an issue.

Maybe adding something like "Please install XCode or the XCode command line tools." would be enough bait.

Reply:
Oh, it looks like you are also logging to the output window.  Perhaps redirecting them there would be good here instead.